### PR TITLE
Feat/new icon and popup

### PR DESCRIPTION
**Pull request by zatteo**
- Add a new icon (mabulle.png)
- Add a modal displayed only on Android to inform the user that the icon has changed
- Use flags to allow icon change and set default icon remotely

image

#### Checklist

Before merging this PR, the following things must have been done:

* [ ] Faithful integration of the mockups at all screen sizes
* [x] Tested on iOS
* [x] Tested on Android
* [ ] Localized in English and French
* [x] All changes have test coverage
* [x] Updated README & CHANGELOG, if necessary

### CODE REVIEW

File: src/libs/icon/icon.spec.ts [hunk 1]
```diff
@@ -1,21 +1,50 @@
...
+  'flagship.icon.defaultIcon': 'cozy'
+}
+
+const ICON_CHANGE_NOT_ALLOWED_FLAGS = {
+  'flagship.icon.changeAllowed': false
+}
+
+const ICON_CHANGE_UNDEFINED_FLAGS = {
+  'flagship.icon.changeAllowed': false
+}
```
Comment (Crash--) [inline src/libs/icon/icon.spec.ts:27]: are they documented in the wiki?

File: patches/react-native-change-icon+4.0.0.patch [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-             promise.reject("ICON_INVALID");
+@@ -83,6 +83,7 @@ public class ChangeIconModule extends ReactContextBaseJavaModule implements Appl
              return;
          }
-+        this.classesToKill.clear();
          this.classesToKill.add(this.componentClass);
++        this.classesToKill.remove(activeClass); // remove the new icon from the list of classes to kill to avoid being unable to start the app
          this.componentClass = activeClass;
          activity.getApplication().registerActivityLifecycleCallbacks(this);
+         iconChanged = true;
```
Comment (Crash--) [inline patches/react-native-change-icon+4.0.0.patch:12]: is is necessary this var?

**Review by Crash-- — APPROVED** [review]

File: patches/react-native-change-icon+4.0.0.patch (hunk 2 above)
Comment (zatteo) [reply patches/react-native-change-icon+4.0.0.patch:12]: This var is not added in "my" patch. I updated the patch package 1 line below the previous patch package, so it took one line more at then end.

File: src/libs/icon/icon.spec.ts (hunk 1 above)
Comment (zatteo) [reply src/libs/icon/icon.spec.ts:27]: Now, yes 😄

**Pull request merged**